.storybook/preview.tsx

```diff
@@ -22,6 +22,7 @@ import type { Preview } from '@storybook/react';
 
 import '../packages/bpk-stylesheets';
 import '../packages/bpk-stylesheets/font';
+import '../packages/bpk-stylesheets/base.css';
```

Comment:
This might not be needed now as a recent PR #3244 should now be imported on the first line above L23

So here we would be importing this file twice